lambda_handlers/docker_authorizer.py

```diff
@@ -0,0 +1,84 @@
+'''
+    Fetch docker token from registry for hub pull and push operation for docker hub
+    Token can be alternately obtained by curl:
+    curl "https://auth.docker.io/token?service=registry.docker.io&scope=repository:jina-ai/jina:pull,push"
+'''
+import re
+import logging
+import urllib3
+import requests
```

Comment:
why do we need `requests` module? we can use `urllib3.PoolManager()`. In Lambda, for using non-standard modules, we need to build a deployment package & upload.

Reply:
done